### ISSUE TITLE
unify "rich workspaces" and "README.md" extension

### ISSUE THREAD

**Issue by brainchild0** [nextcloud/text#1213]
Presently the following two approaches are available for augmenting folder display with documentation.

1. "Rich workspaces", provided by the Text application.
2. The [*README.md*](https://apps.nextcloud.com/apps/files_readmemd) application.

Rich workspaces has an elegant implementation and carries most of the functionality of *README.md*, but also imposes the substantial constraint of automatically generated empty *readme* files, unless disabled completely, for a user or site-wide (see #586).

User experience might improve if rich workspaces replaced the automatic functionality of creating these files, in preference to an action available for   manual invocation by the user. When invoked, the action would open an editor whose buffer would be saved to a new *readme* file.

In any case whatsoever, user experience would improve if automatic creation of empty *readme* files were not a default behavior.

Furthermore, the overlapping functionality of the two applications, with *Readme.md* having an explicit option for disabling rich workspaces, pollutes the ecosystem and incurs a risk of various kinds of conflicts, both of the technical kind with respect to operating the software, and of the social kind   with respect to   the community agreeing on a preferred approach to a common problem.

**Comment by Pilzinsel64** [comment]
I agree, the automatically generated empt readme files can be annoying. An alternative that I think (as an end user) is to only create  and keep the readme file if it's not empty. If there is no input in the text editor, just remove it or don't create it.

**Comment by brainchild0** [comment]
My idea had been that the application would provide a user with an action to create or to remove a *readme* file. The idea presented by @Pilzinsel64  is a second possibility. In this case, I wonder, if a user deletes all content from an existing, populated file, whether the application would automatically remove the file as well. In either case, the change in behavior would offer a considerable improvement to usability.

**Comment by max-nextcloud** [comment]
The empty entry that started the readme has been replaced with a button in the :heavy_plus_sign: dialogue.
I think that covers the feature in the description.
It does not address the different extensions. But I don't think there's anything to do here.
